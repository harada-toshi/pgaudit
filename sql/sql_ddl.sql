\! cp sql/sql_ddl/sql_ddl.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/sql_ddl/sql_ddl.sql
\! rm /tmp/config.conf
\! grep "AUDIT:" ./tmp_check/tmp_check.log > results/sql_ddl.out

