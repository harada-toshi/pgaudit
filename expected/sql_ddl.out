	        format = 'AUDIT: %class,%command_tag,%object_type,%object_id,%command_text'
LOG:  AUDIT: DDL,CREATE TABLE,TABLE,public.foo,CREATE TABLE foo (
LOG:  AUDIT: DDL,CREATE INDEX,INDEX,public.foo_id_key,CREATE TABLE foo (
LOG:  AUDIT: DDL,CREATE TABLE AS,TABLE,public.bar,CREATE TABLE bar AS SELECT * FROM foo;
LOG:  AUDIT: DDL,ALTER TABLE,TABLE,public.baz,ALTER TABLE foo RENAME TO baz;
LOG:  AUDIT: DDL,DROP TABLE,TABLE,public.bar,DROP TABLE bar;
LOG:  AUDIT: DDL,DROP TABLE,TABLE,public.baz,DROP TABLE baz;
LOG:  AUDIT: DDL,DROP TABLE,TABLE CONSTRAINT,foo_id_key on public.baz,DROP TABLE baz;
LOG:  AUDIT: DDL,DROP TABLE,INDEX,public.foo_id_key,DROP TABLE baz;
